client/pdo/client/scripts/eservicedatabaseCLI.py

```diff
@@ -0,0 +1,271 @@
+        db.load_database(config['Service']['EnclaveServiceDatabaseFile'], merge = True)
+        
+        num_removed = 0 
+
+        if config['Service'].get('EnclaveServiceNames'):
+            names = config['Service']['EnclaveServiceNames']
+            for name in names:
+                num_removed += db.remove_info_from_database(name = name)
+        
+        if config['Service'].get('EnclaveServiceURLs'):
+            urls = config['Service']['EnclaveServiceURLs']
+            for url in urls:
+                num_removed +=  db.remove_info_from_database(url = url)
+
+        # save as json file
+        try:
+            db.save_database(config['Service']['EnclaveServiceDatabaseFile'], overwrite = True)
+        except Exception as e:
+            logger.error('Unable to Remove entries: Failed to save as json file %s', str(e))
+            sys.exit(-1)
+
+        logger.info('Removed %d  entries from the database', num_removed)
+    
+    else:
+        logger.error('Unsupported database command')
+        sys.exit(-1)
+    
+    sys.exit(0)
+
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## DO NOT MODIFY BELOW THIS LINE
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+ContractHost = os.environ.get("HOSTNAME", "localhost")
+ContractHome = os.environ.get("PDO_HOME") or os.path.realpath("/opt/pdo")
+ContractEtc = os.path.join(ContractHome, "etc")
+ContractKeys = os.path.join(ContractHome, "keys")
+ContractLogs = os.path.join(ContractHome, "logs")
+ContractData = os.path.join(ContractHome, "data")
+LedgerURL = os.environ.get("PDO_LEDGER_URL", "http://127.0.0.1:8008/")
+ScriptBase = os.path.splitext(os.path.basename(sys.argv[0]))[0]
+
+config_map = {
+    'base' : ScriptBase,
+    'data' : ContractData,
+    'etc'  : ContractEtc,
+    'home' : ContractHome,
+    'host' : ContractHost,
+    'keys' : ContractKeys,
+    'logs' : ContractLogs,
+    'ledger' : LedgerURL
+}
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+def Main() :
+    # parse out the configuration file first
+    conffiles = [ 'pcontract.toml' ]
+    confpaths = [ ".", "./etc", ContractEtc ]
+
+    parser = argparse.ArgumentParser()
+    
+    parser.add_argument('--config', help='configuration file', nargs = '+')
+    parser.add_argument('--config-dir', help='configuration file', nargs = '+')
+    parser.add_argument('--command', help='database command to execute : Chose one from {create, add, remove, update}', required=True, type=str)
+    parser.add_argument('--eservice-url', help='service urls',  nargs='+')
```

Comment:
did you look at the pattern used in the pdo shell? that pattern is far more standard for python. the command is bareword (no switch) and you can use per-command argument processing with subparsers.

Reply:
yes, i agree that for eservice it makes sense, but this creates issues with backward compatibility. all existing tests will have to be modified, so decided to leave it untouched for now